src/amo/actions/reviews.js

```diff
@@ -268,3 +271,58 @@ export const hideReplyToReviewForm = (
 ): HideReplyToReviewFormAction => {
   return reviewIdAction({ type: HIDE_REPLY_TO_REVIEW_FORM, reviewId });
 };
+
+type FlagReviewParams = {|
+  errorHandlerId: string,
+  note?: string,
+  reason: FlagReviewReasonType,
+  reviewId: number,
+|};
+
+export type FlagReviewAction = {|
+  type: typeof SEND_REVIEW_FLAG,
+  payload: FlagReviewParams,
+|};
+
+export const flagReview = (
+  { errorHandlerId, note, reason, reviewId }: FlagReviewParams = {}
+): FlagReviewAction => {
+  if (!errorHandlerId) {
```

Comment:
We should really have a helper for this, heh.

Reply:
I filed a bug for it a while ago https://github.com/mozilla/addons-frontend/issues/1461